bin/brew

```diff
@@ -45,5 +50,66 @@ fi
 
 export HOMEBREW_BREW_FILE
 export HOMEBREW_RUBY_PATH
+export HOMEBREW_PREFIX
+export HOMEBREW_REPOSITORY
+export HOMEBREW_LIBRARY
 
-exec "$HOMEBREW_RUBY_PATH" -W0 "$BREW_LIBRARY_DIRECTORY/brew.rb" "$@"
+if [ $(uname -s) = "Darwin" ]
```

Comment:
And in any case please don't omit the quotes around `$(uname -s)` for the sake of safety and consistency.